brainstorm/src/components/items.tsx

```diff
@@ -78,15 +98,14 @@ export function ItemsView(props: {
 	}
 }
 
-/**
- * Removes a node from its parent {@link Items}.
- * If the note is not in an {@link Items}, it is left unchanged.
- */
-export function deleteItem(item: Item): void {
+export interface ItemParent {
+	deleteItem(item: Item): void;
+}
+
+export function deleteItemFromParent(item: Item & TreeNode): void {
```

Comment:
Item extends TreeNode already
```suggestion
export function deleteItemFromParent(item: Item): void {
```

Reply:
Ah, I missed that when I replaced the type. I think I had ItemParent originally.